### PR TITLE
Spelling Fixes

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes spelling errors.

## Why It's Good For The Game
Improves user experience.

## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: maps/submaps/surface_submaps/mountains/quarantineshuttle.dmm [hunk 1]
```diff
@@ -35,7 +35,7 @@
...
...acu-<BR><B>06:36:50: (Sound of emergency shutters closing)</B><BR><B>Captain Willis 06:36:51: </B>What the hell? Control we just had a remote activation of our emergency shutters, please advise.<BR><B>V.I.S Traffic Control 06:38:10:</B> Captain, please tune to frequency 1493.8 we are passing you on to local emergency response units. Godspeed.<BR><B>Captain Willis 06:38:49:</B> This is Captain Willis of Major Bill's Transportation flight MBT-540 we have eighteen souls aboard and our emergency lockdown shutters have engaged remotely. Do you read?<BR><B>S.D.D.C:</B> This is the Sif Department of Disease Control, your vessel has been identified as carrying highly sensitive materials, and due to the nature of your system's automated alerts you will be asked to remain in quarantine until we are able to determine the nature of the pathogens aboard and whether it has entered the air circulation system. Please remain in your cockpit at this time.<BR> ** </BR><B>Captain Adisu 17:23:58:09:</B> I don't think they're opening those doors Ted. I don't think they're coming.</BR>"; item_state = "paper"; name = "Black Box Transcript MBT-540"},/turf/simulated/shuttle/floor,/area/submap/cave/qShuttle)
```
Comment (EvilJackCarver) [inline maps/submaps/surface_submaps/mountains/quarantineshuttle.dmm]: What's the change in this line? I'm not seeing it on a skim, and Github isn't highlighting it.

Comment (klorpa) [reply maps/submaps/surface_submaps/mountains/quarantineshuttle.dmm]: recieving -> receiving

File: code/modules/busy_space/organizations.dm [hunk 2]
```diff
@@ -465,7 +465,7 @@
 /datum/lore/organization/tsc/independent
 	name = "Free Traders"
 	short_name = "Free Trader"
-	desc = "Though less common now than they were in the decades before the Sol Economic Organization took power, independent traders remain an important part of the galactic economy, owing in no small part to protective tarrifs established by the Free Trade Union in the late twenty-forth century."
+	desc = "Though less common now than they were in the decades before the Sol Economic Organization took power, independent traders remain an important part of the galactic economy, owing in no small part to protective tariffs established by the Free Trade Union in the late twenty-forth century."
```
Comment (EvilJackCarver) [inline code/modules/busy_space/organizations.dm]: "twenty-forth century?"

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 3]
```diff
@@ -29,13 +29,13 @@
...
 	desc = "Decreases your susceptibility to electric shocks by a 25% amount."
-	cost = 2 //This effects tasers!
+	cost = 2 //This effects Tasers!
 	var_changes = list("siemens_coefficient" = 0.75)
 
 /datum/trait/nonconductive_plus
 	name = "Major Non-Conductive"
 	desc = "Decreases your susceptibility to electric shocks by a 50% amount."
-	cost = 3 //Let us not forget this effects tasers!
+	cost = 3 //Let us not forget this effects Tasers!
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm]: Affect versus effect

(It had adverse effects / it adversely affected things)

**Pull request merged**